plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergAvroConnectorTest.java

```diff
@@ -49,4 +49,10 @@ public void testIncorrectIcebergFileSizes()
     {
         throw new SkipException("Avro does not do tail reads");
     }
+
+    @Override
+    protected boolean isFileSorted(String path, String sortColumnName)
+    {
+        throw new SkipException("Unimplemented");
```

Comment:
see comment in FTE test. throw unsupported and override selected methods

also, worth having a TODO + issue?

Reply:
or, maybe sorting Avro files has no sense (has it?), so we should skip doing that.
why add latency for the writer, if the reader not gonna leverage that?